testnets/deardogetestnet/chain.json

```diff
@@ -0,0 +1,103 @@
+{
+  "$schema": "../../chain.schema.json",
+  "chain_name": "deardogetestnet",
```

Comment:
please rename chain_name (in both files) and the directory to doravotatestnet.

Reply:
These are two different testnets, so we need to keep both deardogetestnet and doravotatestnet

Reply:
Sorry, I'd forgotten there was already a doravotatestnet when I commented. But I presume it's still a testnet for doravota, so I think in this case it should be doravotatestnet2, a convention we see with many other chains with multiple testnets

Reply:
Sorry, I don't see how this is resolved. The git_repo and genesis_url still point to dora vota, so it still looks like this should be doravota2...

Reply:
I fixed this problem. @JeremyParish69 
https://github.com/cosmos/chain-registry/pull/3213/files#diff-14bbdcbd15770599c05050af74677168f99c7bf946a103deaf11139e23d365e3R29-R34